internal/reconcile/prometheus_rule.go

```diff
@@ -36,7 +36,6 @@ func PrometheusRule(er record.EventRecorder, k8Client client.Client, desired *mo
 			return nil
```

Comment:
Any reason to not completely remove this file?

Reply:
Do we anticipate needing to create prometheus rules in the future? This seemed generic enough to keep for now for the future is my thinking.